open-telemetry/otel-collector.mdx

```diff
@@ -0,0 +1,133 @@
+
+The following document describes how to natively configure the OpenTelemetry Collector Contrib distribution with the Middleware platform. Use the OpenTelemetry Collector to collect, process, and export your telemetry data that is not accessable through the core OpenTelemetry Project.
+
+<Note> The Kubernetes OpenTelemetry Collector Contrib distribution is not available and in development. </Note>
+
+# Linux Installation
+
+<Accordion title="Step 1: Install DEB or RPM">
+
+    [Download](https://opentelemetry.io/docs/collector/installation/#linux) the `opentelemetry-collector` with the same package manager you used to install the MW Agent.
+</Accordion>
+
+<Accordion title="Step 2: Use otelcol-contrib">
+    Use the `otelcol-contrib (opentelemetry-collector-contrib)` distribution instead of the `otelcol (opentelemetry-collector)` for the OpenTelemetry Collector.
+
+    When prompted to download `otelcol`, download one of the following instead:
+
+    <CodeGroup>
+
+    ```powershell DEB
+    # AMD64
+    otelcol-contrib_0.92.0_linux_amd64.deb
+    # ARM64
+    otelcol-contrib_0.92.0_linux_arm64.deb
+    ```
+
+    ```powershell RPM
+    # AMD64
+    otelcol-contrib_0.92.0_linux_amd64.rpm 
+    # ARM64
+    otelcol-contrib_0.92.0_linux_arm64.rpm
+    ```
+
+    </CodeGroup>  
+</Accordion>
+
+<Accordion title="Step 3: Follow OTel Config">
+    Once you have installed Linux, proceed to the [Configuration Section](/open-telemetry/otel-collector#configuration).
+</Accordion>
+
+# Docker Installation
+
+<Accordion title="Step 1: Install DockerHub or ghcr.io">
+    [Download](https://opentelemetry.io/docs/collector/installation/) the opentelemetry-collector with the same container registry you used to install the MW Agent.
+</Accordion>
+
+<Accordion title="Step 2: Use otelcol-contrib">
+    Ensure the container image refers to the `opentelemetry-collector-contrib` and not core `opentelemetry-collector`.
+</Accordion>
+
+<Accordion title="Step 3: Follow OTel Config">
+    Once you have installed Docker, proceed to the [Configuration Section](/open-telemetry/otel-collector#configuration).
+</Accordion>
+
+# Configuration
+
+<Accordion title="Step 1: Configure Collector">
+    Follow the [OTel Configuration Instructions](https://opentelemetry.io/docs/collector/configuration/) to configure the `opentelemetry-collector-contrib`
+
+    <Note> You must fully configure the OTel Collector before adding more code to the `processors`, `exporters` and `service` section. </Note>
+</Accordion>
+
+<Accordion title="Step 2: Add to Processors">
+
+    Add the following code to the `processors` section:
+
+    ```YAML YAML
+    processors:
+    resourcedetection:
+        detectors: [ env, system ]
+        timeout: 2s
+        system:
+        hostname_sources: [ os ]
+    resource/middleware:
+        attributes:
+        - action: upsert
+        key: mw.account_key
+        value: "<your-mw-api-key>"
+        - action: upsert
+        from_attribute: host.name
```

Comment:
spacing on these two actions is off. The two attributes need to have proper spacing. See Tejas' doc for deets